tests/integration/prerelease.bats

```diff
@@ -168,3 +168,28 @@ teardown() {
     unset GITHUB_RUN_ID
     unset GITHUB_RUN_NUMBER
 }
+
+@test "minimal increment prerelease multiple times w/ prefix" {
+    avakas_wrapper bump "$REPO" patch --prerelease --prerelease-prefix beta
+    avakas_wrapper show "$REPO"
+    [ "$output" == "0.0.1-beta.1" ]
+    avakas_wrapper bump "$REPO" patch --prerelease --prerelease-prefix beta
+    avakas_wrapper show "$REPO"
+    [ "$output" == "0.0.1-beta.2" ]
+}
+
+
+@test "increment prerelease multiple times w/ prefix, changing prefix" {
```

Comment:
The tests don't cover the multi-pre-release-prefix-increment case that's mentioned in `_get_prerelease_version`

Reply:
right you are

Reply:
actually, that behavior isn't fully supported by the CLI (adding nargs='+') to prerelease-prefix gets us close.
The internals currently accept either a string or a sequence of strings (well, collection, but using an unordered collection would be....odd), but the CLI only supports a single string. 

if fully supporting that (it's valid SemVer, fwiw) is desired, I could do that work, probably in another PR?

Reply:
I also would like to add a unit test here and there,  this little section is probably a hot target for that

Reply:
_actual_ unit tests would be rad to have at some point

Reply:
don't threaten me with a good time

Reply:
I _believe_ all the test cases we currently care about are captured in this bats test. Created https://github.com/otakup0pe/avakas/issues/75 to track switching up our testing a bit more at some point in the future

Reply:
I'm not sure what the solution to @colakong's comment is tho

Reply:
Maybe we can ignore that case and say that it's unsupported; If a pre-release label is being incremented, it just targets the first one if more than one exist.

Reply:
Yeah if we aren't going to implement it now/ever that should be mentioned in the README.

Reply:
I think it would be fairly trivial to implement (make `--prerelease-prefix` take a dotted string and/or more nargs = '+'). Since the prerelease segments were already handled as tuples internally, I implemented against that before checking the CLI

Reply:
There may need to be a bit more work in how the prereleases are done (I realized this wasn't covered in the tests when I went to go test this behavior), but it's close.